src/theme/material/variants/default.m.css

```diff
@@ -3,6 +3,7 @@
 
 	/* variables taken from @material/theme/dist/mdc.theme.css */
 	--mdc-theme-primary: #6200ee;
+	--mdc-theme-dark-primary: #bb86fc;
```

Comment:
could you name this `--mdc-theme-primary-on-dark` perhaps?

Reply:
@tomdye why? Is that the convention?

Reply:
edited above

Reply:
@tomdye thanks for updated the comment.

Reply:
@tomdye updated.